CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * MapboxCoreNavigation now depends on [MapboxNavigationNative v78._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/78.0.0). ([#3488](https://github.com/mapbox/mapbox-navigation-ios/pull/3488))
 
+### Banners and guidance instructions
+
+* Fixed Night Style for guidance instructions. ([#3503](https://github.com/mapbox/mapbox-navigation-ios/pull/3503))
```

Comment:
Can you explain in a sentence or two what got fixed, both here and in the PR description? Thanks!

Reply:
The second point might be worth calling out if we think the redundant constraints were causing a performance issue:

```suggestion
* `InstructionsCardViewController` now adapts to `NightStyle`. ([#3503](https://github.com/mapbox/mapbox-navigation-ios/pull/3503))
* Fixed an issue where `InstructionsCardViewController` installed duplicate Auto Layout constraints on each location update. ([#3503](https://github.com/mapbox/mapbox-navigation-ios/pull/3503))
```